plugins/catalog-backend/src/modules/core/deprecated.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
```

Comment:
Can these go in the root deprecated.ts instead? It's kinda nice as a dumping ground for ALL deprecated things and then easy deletion if a 2.0 release happens some time in the future.

Reply:
We can do that. The only downside is that we would need to replace the `*` imports at the chain up.
Or we could handle the `deprecated.ts` files similar to `index.ts` files and include only the root `deprecated.ts` at the root `index.ts`.

Reply:
Yeah, it's easier for maintenance if we have all the deprecated properties in one file which is why we'd like to move these to  the root `deprecated.ts`

Reply:
done